packages/react-ui-core/src/List/__tests__/List-test.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import theme from './mocks/theme'
+import ThemedList from '../List'
+
+const List = ThemedList.WrappedComponent
+
+describe('List', () => {
+  let wrapper
+
+  beforeEach(() => {
+    wrapper = mount(
+      <List />,
+    )
+  })
+
+  it('renders a List', () => {
+    wrapper.setProps({ theme })
+    expect(wrapper.find(List).length).toBe(1)
+  })
+
+  it('applies a theme classname', () => {
+    wrapper.setProps({ theme })
+    const className = wrapper.find('ul').prop('className')
+    expect(className).toContain('List')
```

Comment:
`toContain` here will match `List`, `List-orientation` and anything else with List in it, even `Foo-List-Foo`.  Are you sure you want to test this?